src/spdx/parser/jsonlikedict/dict_parsing_functions.py

```diff
@@ -12,7 +12,7 @@
 
 from spdx.model.spdx_no_assertion import SpdxNoAssertion
 from spdx.model.spdx_none import SpdxNone
-from spdx.model.typing.constructor_type_errors import ConstructorTypeErrors
+from common.typing.type_checks import ConstructorTypeErrors
```

Comment:
Why are `ConstructorTypeErrors` imported from `type_checks` now? Does this even work?

Reply:
Good catch! Fixed this.